### PR TITLE
fixed a minor translation mistake

### PR DESCRIPTION
**Pull request by soomin-jeong**
Changes in this pull request:
- Previously it was translated as 'loading *dynamic* data', which was supposed to be 'loading *static* data'.

### CODE REVIEW

**Comment by magul** [comment]
Hi @soomin-jeong, first of all thank You for your pull requests.

Have You seen also that one? #1097 Can You review it? Is it ok?

Also it would be great if You could apply your fixes also on Crowdin here: https://crowdin.com/project/django-girls-tutorial/ko#

**Comment by soomin-jeong** [comment]
Right on it. I am gradually commenting the commits on #1097. I would highly appreciate if you could offer me some details upon how I can apply my fixes on Crowdin. Thanks!

**Comment by magul** [comment]
It's really simple there. You can login using Github account, after that You just need to join translation project here: https://crowdin.com/project/django-girls-tutorial and You're good to start working on translation.

General idea is that we can track there any changes inside source (English) language, so we can see what coverage that we have in any particular language with the top of Github `/en/` directory.

I see that Korean translation there has only 43% coverage, so one of two happened:
 * Korean translations were separately maintain here (on Github) an there (on Crowdin),
 * Gihub-based Korean translation was moved to Crowdin and and wasn't synced since and there are lot changes in English version, so coverage drops (it's unlikely, as I don't recall any way to sync target languages to Crowdin).

**Comment by soomin-jeong** [comment]
Thank you for the kind instructions. I will look through the crowdin. First things first, I have approved the commits on #1097, with some suggestions. Please take a look and feel free to fix and merge them.

**Comment by sujinleeme** [comment]
@corazzon  크로스 체킹 부탁드려요 :)

File: ko/css/README.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ W3Schools에서 [CSS 선택자](https://www.w3schools.com/cssref/css_selectors.a
 {% load staticfiles %}
 ```
 
-여기에서 동적 파일을 로딩하는 거에요. :)
+여기에서 정적 파일을 로딩하는 거에요. :)
```
Comment (sujinleeme) [inline ko/css/README.md]: Thanks for correcting wrong translation.

**Comment by soomin-jeong** [comment]
@sujinleeme 안녕하세요! 반갑습니다. 많은 도움 받았습니다. 가능하면 더 검토해볼게요!

**Comment by sujinleeme** [comment]
@soomin-jeong Thanks for your contribution and I checked there's wrong translation as you sent PR. Therefore you don't need to close issue & PR. :) @RachellCalhoun will help you.

**Comment by ekohl** [comment]
I'm very unfamiliar with the translation workflow. Do we have a guide for maintainers on how to deal with this?

**Comment by sujinleeme** [comment]
I wrote django girls slack many days ago. I put a lot of my effort on django girls korean translation when it started. After finished and sent pull request to extended tutorial translation, me and another contributor are included in django girls github team. However today, I’ve checked there are many issues regarding korean translation on github repo(because I wasn’t tagged so far). For maniantaince work, using crowding won't  be helpful. What about make a list of main translators each language.

**Comment by soomin-jeong** [comment]
Got lost while figuring out what to do with Crowdin :( . With that said, I would love to help translation and reviewing the PRs in any other possible ways.

File: ko/css/README.md (hunk 1 above)
Comment (sujinleeme) [reply ko/css/README.md]: Yeah, it must to be changed.

**Pull request merged**